Scrabble.Bot/Scrabble.fs

```diff
@@ -109,13 +132,18 @@ module Scrabble =
                 aux st'
             | RCM (CMPlayed (pid, ms, points)) ->
                 (* Successful play by other player. Update your state *)
-                let st' = State.mkState st.board st.dict st.playerId st.hand st.players pid // This state needs to be updated
-                aux st'
-            | RCM (CMPlayFailed (pid, ms)) ->
-                (* Failed play. Update your state *)
-                let st' = State.mkState st.board st.dict st.playerId st.hand st.players pid // This state needs to be updated
+                let placedTiles = List.map (fun (coord, (_ , tile)) -> (coord, tile)) ms
+                let st' = st 
+                                    |> State.placeLetters (Seq.ofList placedTiles) 
+                                    |> State.addPoints pid points

```

Comment:
I don't think it makes sense to call the method here, since it won't do anything??

Reply:
It is becuase i can see that you have a task with the purpose to keep track of all the other players points. For that reason i created a general purpose so that you could implement the rest